tests/pytests/test_short_read.py

```diff
@@ -21,37 +21,24 @@
 SHORT_READ_BYTES_DELTA = int(os.getenv('SHORT_READ_BYTES_DELTA', '1'))
```

Comment:
what did you do here?

Reply:
removed duplications, paired up RDBs and `ExpectedIndex` values (so we don't need to rely on `zip` calls), and finally added dynamic test generation for each RDB file in the dictionary